demo/index.html

```diff
@@ -18,46 +18,80 @@
     <title>iron-autogrow-textarea demo</title>
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
     <link rel="import" href="../iron-autogrow-textarea.html">
-    <link rel="import" href="../../paper-styles/demo-pages.html">
+
+    <style is="custom-style" include="demo-pages-shared-styles">
+      iron-autogrow-textarea {
+        display: block;
+        width: 200px;
+        margin: 5px 0;
+      }
+
+      textarea {
+        width: 200px;
+      }
+
+      .vertical-section {
+        box-sizing: border-box;
+        width: 400px;
```

Comment:
is there a reason why changing this to 400px?
You could set the `centered` class to `body` which will set its width to 500px, and you'll have 1 div less (like in [paper-toast](https://github.com/PolymerElements/paper-toast/blob/master/demo/index.html#L31)